server/controllers/AuthController.js

```diff
@@ -111,6 +112,43 @@ class AuthController {
       });
     }).catch(next);
   }
+
+  /**
+ * Update password for logged in users
+ * @param {*} req - Request object
+ * @param {*} res - Response object
+ * @param {*} next - Next function
+ * @returns {token} token - JWT token
+ */
+  static updatePassword(req, res, next) {
+    const { oldPassword, password } = req.body;
+
+    User.findOne({
+      where: {
+        id: req.decoded.userId
+      }
+    }).then((user) => {
+      if (!bcrypt.compareSync(oldPassword, user.password)) {
+        return res.status(401).json({
+          message: 'password incorrect, try again'
+        });
+      }
+      if (bcrypt.compareSync(password, user.password)) {
+        return res.status(409).json({
+          message: 'New password is the same as previous password'
+        });
+      }
+      const hashPassword = bcrypt.hashSync(password, salt);
```

Comment:
Check `utilities.js` for the helper function to hash password

Reply:
Done, thanks